builtin/settingtypes.txt

```diff
@@ -601,6 +612,24 @@ enable_auto_exposure (Enable Automatic Exposure) bool false
 #    Requires: enable_post_processing, enable_auto_exposure
 exposure_compensation (Exposure compensation) float 0.0 -1.0 1.0
 
+#   Set the post processing gamma value.
+#   Higher values give lower contrast and vice versa.
+#   Range: from 1.0 to 5.0
+#   Default: 1.6
+#
+#   Requires: enable_post_processing, tone_mapping
+secondstage_gamma (Gamma) float 1.6 1.0 5.0
+
+#   Apply ASL CDL color grading to make brighter colors warmer and darker colors cooler.
+#
+#   Requires: enable_post_processing
+enable_color_grading (Color grading) bool false
+
+#   Apply vignette effect to darken the edges of the screen.
+#
+#   Requires: enable_post_processing
+enable_vignette (Vignette) bool false
```

Comment:
Why have a client flag at all? Make it only server controlled so that games can rely on it.